### DISCUSSION TITLE
Start up problem

### DISCUSSION THREAD

**Discussion by edwin123456** [Koenkk/zigbee2mqtt#14030] [General]
Zigbee2mqtt doesnt start
See:
Started zigbee2mqtt.
sep 16 17:00:04 edwin npm[14475]: > zigbee2mqtt@1.25.1 start
sep 16 17:00:04 edwin npm[14475]: > node index.js
sep 16 17:00:05 edwin npm[14475]: (node:14488) UnhandledPromiseRejectionWarning: YAMLException: bad indentation of a mapping entry (24:3)
sep 16 17:00:05 edwin npm[14475]:  21 | # Serial settings
sep 16 17:00:05 edwin npm[14475]:  22 | serial: __0X00124B001949B253
sep 16 17:00:05 edwin npm[14475]:  23 |   # Location of the adapter (see  ...
sep 16 17:00:05 edwin npm[14475]:  24 |   port: /dev/ttyACM0
sep 16 17:00:05 edwin npm[14475]: --------^
sep 16 17:00:05 edwin npm[14475]:  25 | # Will run frontend on port 8080
sep 16 17:00:05 edwin npm[14475]:  26 | frontend: true
sep 16 17:00:05 edwin npm[14475]:     at generateError (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:183:10)
sep 16 17:00:05 edwin npm[14475]:     at throwError (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:187:9)
sep 16 17:00:05 edwin npm[14475]:     at readBlockMapping (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:1182:7)
sep 16 17:00:05 edwin npm[14475]:     at composeNode (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:1441:12)
sep 16 17:00:05 edwin npm[14475]:     at readDocument (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:1625:3)
sep 16 17:00:05 edwin npm[14475]:     at loadDocuments (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:1688:5)
sep 16 17:00:05 edwin npm[14475]:     at Object.load (/opt/zigbee2mqtt/node_modules/js-yaml/lib/loader.js:1714:19)
sep 16 17:00:05 edwin npm[14475]:     at Object.read (/opt/zigbee2mqtt/lib/util/yaml.ts:7:21)
sep 16 17:00:05 edwin npm[14475]:     at read (/opt/zigbee2mqtt/lib/util/settings.ts:310:20)
sep 16 17:00:05 edwin npm[14475]:     at getInternalSettings (/opt/zigbee2mqtt/lib/util/settings.ts:406:21)
sep 16 17:00:05 edwin npm[14475]: (Use `node --trace-warnings ...` to show where the warning was created)
sep 16 17:00:05 edwin npm[14475]: (node:14488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
sep 16 17:00:05 edwin npm[14475]: (node:14488) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
sep 16 17:00:05 edwin systemd[1]: zigbee2mqtt.service: Succeeded.
sep 16 17:00:05 edwin systemd[1]: zigbee2mqtt.service: Service RestartSec=100ms expired, scheduling restart.
sep 16 17:00:05 edwin systemd[1]: zigbee2mqtt.service: Scheduled restart job, restart counter is at 14.
sep 16 17:00:05 edwin systemd[1]: Stopped zigbee2mqtt.
sep 16 17:00:05 edwin systemd[1]: Started zigbee2mqtt.

**Comment by Koenkk** [comment]
Your YAML is not formatted correctly, use http://www.yamllint.com/ to find the issue.
